README.md

```diff
@@ -253,6 +253,8 @@ In method signatures, there should be a space after the method type (-/+ symbol)
 
 The usage of the word "and" is reserved.  It should not be used for multiple parameters as illustrated in the `initWithWidth:height:` example below.
 
+There should be no space after method return type's pair of parenthesis.
```

Comment:
I don't think there's a need to mention the parentheses. "There should be no space between the method's return type and its name." should be sufficient.

Reply:
Well. Probably, too detailed. Will edit.